alea/models/blueice_extended_model.py

```diff
@@ -245,12 +264,20 @@ def _generate_data(self, **generate_values) -> dict:
         data["generate_values"] = dict_to_structured_array(generate_values)
         return data
 
+    def store_data(self, file_name, data_list, data_name_list=None, metadata=None):
```

Comment:
Do you want to change `likelihood_names` here to `data_name_list`:
https://github.com/XENONnT/alea/blob/b710149e7bf2c833ea7d035dcd7416dafeeb2b34/alea/model.py#L214-L218? 

I think the logic is:
1. `ll` and `generate_data` are mandatory for a new model, this is already demonstrated at `GaussianModel`
2. `BlueiceExtendedModel` needs an overwritten `store_data` function because it has an advantage that `generate_values` is also in the `self.data`, so that the `StatisticalModel.store_data` can not be directly applied on `BlueiceExtendedModel`. 
3. Someone overwriting `store_data`, does not violate the truth that only `ll` and `generate_data` are mandatory. In principle, one can overwrite everything.